code/game/objects/items/devices/radio/beacon.dm

```diff
@@ -89,6 +89,10 @@
 	desc = "A label on it reads: <i>Warning: Activating this device will send a high-ordinance EMP explosive to your location</i>."
 	bomb = /obj/machinery/syndicatebomb/emp
 
+/obj/item/radio/beacon/syndicate/bomb/power_sink //Yes it's not a bomb but this code works the way we want
```

Comment:
This feels so wrong. I would rather a version with `/obj/item/radio/beacon/powersink` with a separate attackhand